projectroles/templates/projectroles/_project_sidebar.html

```diff
@@ -1,146 +1,20 @@
-         <span class="sodar-pr-sidebar-icon">
-           <i class="iconify"
-              data-icon="{{ plugin.icon }}"
-              data-height="{{ sidebar_icon_size }}"></i>
-         </span>
-         <br />{{ app_legend | safe }}
-        </a>
-      </li>
-    {% endif %}
-  {% endfor %}
-
-  {# Role and project editing #}
-  {% if can_view_roles %}
-    <li id="sodar-pr-nav-project-roles"
-        class="nav-item sodar-pr-sidebar-nav-item {% get_pr_link_state projectroles_urls request role_urls %}">
-      <a class="nav-link"
-         href="{% url 'projectroles:roles' project=project.sodar_uuid %}"
-         id="sodar-pr-link-project-roles">
-        <span class="sodar-pr-sidebar-icon">
-          <i class="iconify"
-             data-icon="mdi:account-multiple"
-             data-height="{{ sidebar_icon_size }}"></i>
-          <br />Members
-        </span>
-      </a>
-    </li>
-  {% endif %}
-  {% if can_update_project %}
-    <li id="sodar-pr-nav-project-update"
-        class="nav-item sodar-pr-sidebar-nav-item {% get_pr_link_state projectroles_urls request 'update' %}">
-      <a class="nav-link"
-         href="{% url 'projectroles:update' project=project.sodar_uuid %}"
-         id="sodar-pr-link-project-update">
-        <span class="sodar-pr-sidebar-icon">
-          <i class="iconify"
-             data-icon="mdi:lead-pencil"
-             data-height="{{ sidebar_icon_size }}"></i>
-          <br />Update<br />{% get_display_name project.type title=True %}
-        </span>
-      </a>
-    </li>
-  {% endif %}
-
-{% endif %}
-
-{# Project and Category Creation #}
-{% if project and project.type == 'CATEGORY' %}
-  {% has_perm 'projectroles.create_project' request.user project as can_create_project %}
-  {% if allow_creation and can_create_project and not project.is_remote %}
-    <li id="sodar-pr-nav-project-create"
-        class="nav-item sodar-pr-sidebar-nav-item {% get_pr_link_state projectroles_urls request 'create' %}">
-      <a class="nav-link"
-         href="{% url 'projectroles:create' project=project.sodar_uuid %}"
-         id="sodar-pr-link-project-create">
-        <span class="sodar-pr-sidebar-icon">
-          <i class="iconify"
-             data-icon="mdi:plus-thick"
-             data-height="{{ sidebar_icon_size }}"></i>
-        </span>
-        <br />Create<br />{{ project_display }} or<br />{{ category_display }}
-      </a>
-    </li>
-  {% endif %}
-{# Allow project creation under root #}
-{% elif disable_categories and request.user.is_superuser %}
-  <li id="sodar-pr-nav-project-create"
-      class="nav-item sodar-pr-sidebar-nav-item {% get_pr_link_state projectroles_urls request 'create' %}">
+{% get_project_app_links request project as sidebar_links %}
+{% for link in sidebar_links %}
+  <li id="sodar-pr-nav-{{ link.name }}"
+      class="nav-item sodar-pr-sidebar-nav-item {% if link.active %}active{% endif %}"
+      style="word-break: break-word;">
```

Comment:
Add this in `projectroles.css` under `sodar-pr-sidebar-nav-item a`. We should not use `style` tags except in some very special cases where they're absolutely required.